Sources/MapboxNavigation/NavigationCamera.swift

```diff
@@ -34,6 +34,7 @@ public class NavigationCamera: NSObject, ViewportDataSourceDelegate {
     
     func setupGestureRecognizers() {
         makeGestureRecognizersDisableCameraFollowing()
+        makeTapGestureRecognizerStopAnimatedTransitions()
```

Comment:
Because `func setupGestureRecognizers()` already provides the gestureRecognizer to stop the `cameraStateTransition`. Could we just add the `UITapGestureRecognizer` to it and add the `.following` state to the `func stop()`?

Reply:
This would create a misleading behavior. `makeGestureRecognizersDisableCameraFollowing` is meant to 'stop following', which is not the case for tap gesture. Conditioning logic inside this method to distinguish gesture recognizers, current states, etc. just complicates it. Also, users rely on public `stop()` method to cancel any transaction, so it would create a bug if this method would not do that under some conditions.